lcviewernoqt/painters/opengl/sources/gl_font.cpp

```diff
@@ -0,0 +1,248 @@
+#include "gl_font.h"
+#include <QString>
```

Comment:
I need a function which converts (char* / std::string ) --to--> std::wstring (For getting the correct UTF code of extended characters) It will either require some extra lib, but there was already a function in QString

Reply:
Have you tried this: https://stackoverflow.com/a/18597384 ?

Reply:
Yes i have read that, The lib codecvt can be deprecated, i found the most easiest and convenient way by QString, actually i thought why don't use it if we are already building LC3 with Qt. But We can can do with other lib too, i haven't tried that seperately

Reply:
`codecvt` is deprecated in c++17, and we don't use it. There isn't any replacement in std lib, so don't worry, you can use `codecvt` as it's not going to be removed soon

Reply:
Done!! :)